en/installconfig-storage.md

```diff
@@ -82,25 +78,56 @@ boot disk.
 `root_size`: Size of the root partition. Default is 100%, meaning the entire
 size of the root device.
 
-### Bcache Layout
+### LVM layout
 
-A 'Bcache Layout' will create a partition that spans the entire boot disk as
-the backing device. It uses the smallest block device tagged with 'ssd' as the
-cache device. The Bcache device is formatted with ext4 and set as the / mount
-point.  If no block devices exists on the node that are tagged with 'ssd' then
-the Bcache device will not be created and the 'Flat Layout' will be used
-instead:
+The LVM layout creates the volume group `vgroot` on a partition that spans the
+entire boot disk. A logical volume `lvroot` is created for the full size of the
+volume group; is formatted with the ext4 filesystem; and uses the `/` mount point:
 
-| Name      | Size     | Type  | Filesystem Type       | Mount point  |
-|:----------|----------|-------|---------------|--------------|
-| sda       | 100G     | disk  |               |              |
-| sda1      | 512M     | part  | fat32         | /boot/efi    |
-| sda2      | 99.5G    | part  | bc-backing    |              |
-| sdb (ssd) | 50G      | disk  |               |              |
-| sdb1      | 50G      | part  | bc-cache      |              |
-| bcache0   | 99.5G    | disk  | ext4          | /            |
+| Name      | Size        | Type  | Filesystem     | Mount point |
+|:----------|-------------|-------|----------------|-------------|
+| sda       | -           | disk  |                |             |
+| sda1      | 512 MB      | part  | FAT32          | /boot/efi   |
+| sda2      | rest of sda | part  | lvm-pv(vgroot) |             |
+| lvroot    | rest of sda | lvm   | ext4           | /           |
+| vgroot    | rest of sda | lvm   |                |             |
 
-The following options are supported for this layout. :
+The following options are supported:
+
+`boot_size`: Size of the boot partition on the boot disk. Default is 0, meaning not to
+create the boot partition. The '/boot' will be placed on the root filesystem.
+
+`root_device`: The block device to place the root partition on. Default is the boot disk.
+
+`root_size`: Size of the root partition. Default is 100%, meaning the entire size of the
+root device.
+
+`vg_name`: Name of the created volume group. Default is `vgroot`.
+
+`lv_name`: Name of the created logical volume. Default is `lvroot`.
+
+`lv_size`: Size of the created logical volume. Default is 100%, meaning the entire size of
+the volume group.
+
+### Bcache layout
+
+A Bcache layout will create a partition that spans the entire boot disk as the
+backing device. It uses the smallest block device tagged with 'ssd' as the
+cache device. The Bcache device is formatted with the ext4 filesystem and uses
+the `/` mount point. If no block devices exists on the node that are tagged
+with 'ssd' then the Bcache device will not be created and the Flat layout will
+be used instead:
```

Comment:
If there are no 'ssd' tagged block devices on the node, then the Bcache device... ?